packages/block-library/src/query/block.json

```diff
@@ -22,11 +22,19 @@
 				"exclude": [],
 				"sticky": ""
 			}
+		},
+		"layout": {
```

Comment:
This is the attempt for having a structure similar to `__experimentalLayout`. Check PR's description for more info.

Reply:
Should it be experimental then?

Latest Posts block defines two attributes:
https://github.com/WordPress/gutenberg/blob/3aeb89e6fdb8a72741ea9078f199f8265ffb8592/packages/block-library/src/latest-posts/block.json#L39-L46

We can always tweak `providesContext` to support some grouping if that makes it easier:

```json
"providesContext": {
	"queryId": "queryId",
	"query": "query",
	"layout": {
		"columns": "columns",
		"type": "layout"
	}
}
```